### PR TITLE
Release v0.15.1

### PR DESCRIPTION
**Pull request by brimoor**

### CODE REVIEW

**Review by benjaminpkane — APPROVED** [review]
LGTM

**Comment by benjaminpkane** [comment]
Coloring adjustments are ready

**Comment by ehofesmann** [comment]
> Coloring adjustments are ready

It's working great for me! Might be worth throwing some more colors in the default color pool since there are potentially a lot of colors needed for a busy scene. Some overlap is fine, but there were quite a few cases where two instances that were overlapping had the same color.

File: fiftyone/core/config.py [hunk 1]
```diff
@@ -265,11 +274,17 @@ def __init__(self, d=None):
...
+        self.color_by = self.parse_string(
             d,
-            "color_by_value",
-            env_var="FIFTYONE_APP_COLOR_BY_VALUE",
-            default=False,
+            "color_by",
+            env_var="FIFTYONE_APP_COLOR_BY",
+            default="field",
+        )
+        self.color_keypoint_points = self.parse_string(
```
[the remark this answers is not in the document]
Comment (brimoor) [reply fiftyone/core/config.py:283]: Please update the user guide docs as well!

File: fiftyone/core/config.py [hunk 2]
```diff
@@ -375,6 +396,25 @@ def get_colormap(self, colorscale=None, n=256, hex_strs=False):
         return fop.get_colormap(colorscale, n=n, hex_strs=hex_strs)
 
     def _init(self):
+        color_by_value = self.parse_bool(
+            {},
+            "color_by_value",
+            env_var="FIFTYONE_APP_COLOR_BY_VALUE",
+            default=False,
```
Comment (brimoor) [inline fiftyone/core/config.py:403]: I find it unlikely someone is using this, so I personally don't care if this is just silently ignored.

If you do leave this though, setting `default=None` would be a bit better so you can warn if they've hard-coded `False` in their config.

File: fiftyone/core/config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (brimoor) [reply fiftyone/core/config.py:283]: I think this should default to `False`. Pretty sure the typical user would prefer to see the points match the field/instance color.

**Comment by benjaminpkane** [comment]
> I know we can't add too many more colors while still keeping them visually distinct, but we should be able to have more than 12, no?

I'm not sure what colors to add. But I do agree

**Pull request merged**